plugins/dev9ghzdrk/DEV9.cpp

```diff
@@ -40,6 +48,26 @@ HINSTANCE hInst=NULL;
 
 //#define HDD_48BIT
 
+#if defined(__i386__) && !defined(_WIN32)
+
+static __inline__ unsigned long long GetTickCount(void)
+{
+	unsigned long long int x;
+	__asm__ volatile (".byte 0x0f, 0x31" : "=A" (x));
```

Comment:
Don't know. http://www.mcs.anl.gov/~kazutomo/rdtsc.html

Reply:
I'd guess it was compiler/assembler related. I do see a comment in GSdx that says
```
// gcc 4.8 define __rdtsc but unfortunately the compiler crash...
```

It's probably fine to use `__asm__ volatile ("rdtsc" : "=A" (x));` since that's used in at least two other places in the codebase.